backend/lib/database/tables/service_area.py

```diff
@@ -21,3 +21,5 @@ class ServiceArea(Base):
         Geography(geometry_type='POLYGON', srid=4326, spatial_index=True),
         nullable=False
     )
+    state_fips = Column(String, nullable=False)
```

Comment:
should these be indexed?
Shouldnt we add these to rps as well?

Reply:
I don't really see the need to duplicate information between the two tables, since `service_area_id` already links the two.